airflow/contrib/operators/bigquery_operator.py

```diff
@@ -569,3 +570,125 @@ def execute(self, context):
             project_id=self.project_id,
             dataset_id=self.dataset_id
         )
+
+
+class BigQueryDatasetCheckExistOperator(BaseOperator):
+    """"
+    This operator is used to check what a dataset exist in Big Query.
+    https://cloud.google.com/bigquery/docs/reference/rest/v2/datasets/get
```

Comment:
Can you use a `seealso` Sphinx block for all the links you use in any PR? This will make the docs more clean :)

Reply:
@kaxil , thanks for review. I was not sure what need to create separate operators for with, so I agree about leave it only as hook. About Sphinx - okay ))

Reply:
No worries @xnuinside . I really appreciate any time and effort towards making Airflow better :-)

I am still not stripping out the possibility of these operators if there are other users who would have a use-case for this :)